js_modules/dagster-ui/packages/ui-core/src/overview/OverviewSensors.tsx

```diff
@@ -103,12 +103,7 @@ export const OverviewSensors = () => {
       notifyOnNetworkStatusChange: true,
     },
   );
-  const {data: currentData, loading} = queryResultOverview;
-  const data =
-    currentData ??
-    (cachedData?.workspaceOrError.__typename === 'Workspace'
-      ? (cachedData as Extract<typeof cachedData, {workspaceOrError: {__typename: 'Workspace'}}>)
-      : null);
```

Comment:
Removing this for now but will add back a different approach to using the cache in a follow up PR